NuGet.Docs/Consume/Command-Line-Reference.md

```diff
@@ -710,6 +710,107 @@ Specify the API key to save and an optional URL to the server that provided the
 
     nuget setapikey 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -Source http://example.com/nugetfeed
 
+# Folder Repository Commands
+
+
+## Init Command
+
+(v<em>3.3 or above</em>) Adds all the packages from the provided source package source to the destination package source in the following layout instead of making a simple copy of the nupkg file. For this command, only package sources which are folders or UNC shares are allowed. True for both the source and the destination. In order to expand all the files in the package to the destination package source, use the -Expand switch. The following layout has significant performance benefits, when performing a restore or an update against your package source, compared to a folder of nupkg files.
+
+    \\destinationpackagesource\
+        yourpackage\
+            0.0.1-beta\
+                yourpackage.0.0.1-beta.nupkg
+                yourpackage.nuspec
+                yourpackage.0.0.1-beta.nupkg.sha512
+
+### Init Command Usage
+    nuget init <srcPackageSourcePath> <destPackageSourcePath> [options]
+
+Specify the path to source package source to be copied from and the path to the destination package source to be copied to.
+
+### Init Command Options
+<table>
+    <tr>
+        <td>Expand</td>
+        <td>If provided, all the files in the package(s) are added to destination package source.</td>
+    </tr>
+    <tr>
+        <td>Help</td>
+        <td>help</td>
+    </tr>
+    <tr>
+        <td>Verbosity</td>
+        <td>Display this amount of details in the output: normal, quiet, (v<em>2.5</em>) detailed.</td>
```

Comment:
we probably don't need to specify that detailed is only in 2.5 since you can only get this command in 3.3